BAZProject/Commons/UIImageView+.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIImageView+.swift
+//  BAZProject
+//
+//  Created by Manuel S on 05/05/23.
+//
+
+import UIKit
+
+// Metodos extras para la colocacion de las imagenes
+extension UIImageView {
+    
+    /// Consulta desde una url para colocar la imagen
+    func loadImage(url: URL) -> URLSessionDownloadTask {
+        let session = URLSession.shared
+        let downloadTask = session.downloadTask(with: url){[weak self] url, response, error in
```

Comment:
Hay que corregir los espacios entre llaves, paréntesis y corchetes.
```suggestion
        let downloadTask = session.downloadTask(with: url) { [weak self] url, response, error in
```
Lo mismo en la línea 19